src/styles/annotator/adder.scss

```diff
@@ -102,21 +102,19 @@ $adder-transition-duration: 80ms;
   @include focus.outline-on-keyboard-focus;
 
   box-shadow: none;
-  font-family: h;
-  font-size: 18px;
   background: transparent;
   color: var.$grey-mid;
   display: flex;
   flex-direction: column;
   align-items: center;
   border: none;
   cursor: pointer;
+  font-size: 12px;
+  font-family: sans-serif;
+  line-height: 1em;
 
-  padding-top: 10px;
-  padding-bottom: 6px;
-  padding-left: 10px;
-  padding-right: 10px;
-
+  padding: 10px;
+  padding-bottom: 7px;
```

Comment:
This number is a little different because the `line-height` rule eliminates some "padding" underneath the label text.